grafana/grafana-csv-datasource
# website: updated docusaurus packages

updated the software that builds the website.

how to test:

1. go to the `website folder`
2. run `yarn install`
3. run `yarn build`,
4. run `yarn run serve`
5. this will serve the webpage on `http://localhost:3000`. compare the generated website to the one living at https://grafana.github.io/grafana-csv-datasource/ . they should be the same.
